Source/EasyNetQ/Consumer/InternalConsumer.cs

```diff
@@ -123,6 +125,13 @@ private void Cancel()
             if(consumerCancelled != null) consumerCancelled(this, new ConsumerEventArgs(ConsumerTag));
         }
 
+        private void FireConsumerLost()
```

Comment:
OK! I was wrong, the rest of the easynetq seems to have the On* prefix, so I think OnConsumerLost would make the most sense to me, do you agree?

Having Durable=true by default would be my second remark :) 

Thanks a lot for the PR!